example/android/settings.gradle

```diff
@@ -1,2 +1,2 @@
-include ':app', ':react-native-maps'
```

Comment:
This change caused building the example app to fail.  Adding back `':app', ` fixed it.  Once this is changed back, we can merge!